flink-state-backends/flink-statebackend-rocksdb/src/test/java/org/apache/flink/contrib/streaming/state/RocksDBStateBackendConfigTest.java

```diff
@@ -106,6 +106,12 @@ public void testDefaultDbLogDir() throws Exception {
                     RocksDBConfigurableOptions.LOG_LEVEL.defaultValue(),
                     container.getDbOptions().infoLogLevel());
             assertEquals(logFile.getParent(), container.getDbOptions().dbLogDir());
+
+            String longInstancePath = tempFolder.newFolder().getAbsolutePath();
+            while (longInstancePath.length() < 255) {
+                longInstancePath += "/append-for-long-path";
```

Comment:
I'd prefer to use StringBuilder even if compiler could have optimization on '+='.